core/src/main/resources/error/error-classes.json

```
@@ -482,6 +482,12 @@
     ],
     "sqlState" : "22012"
   },
+  "DUPLICATED_MAP_KEY" : {
+    "message" : [
+      "Duplicate map key <key> was found, please check the input data. If you want to remove the duplicated keys, you can set <mapKeyDedupPolicy> to <lastWin> so that the key inserted at last takes precedence."
```

Let's inline the config value `LAST_WIN` as we do for boolean configs, for instance:
```suggestion
      "Duplicate map key <key> was found, please check the input data. If you want to remove the duplicated keys, you can set <mapKeyDedupPolicy> to \"LAST_WIN\" so that the key inserted at last takes precedence."
```
or there is a reason to pass it from the code?

---

I believe there is not a specific reason, so the suggestion sounds reasonable to me.
Let me update it!